nuxeo-drive-client/nxdrive/engine/engine.py

```diff
@@ -1005,3 +1009,29 @@ def fetch_all_users(self):
         except Exception as e:
             log.exception(e)
         return
+
+    def refresh_user(self, user_id):
```

Comment:
I don't know what's the role of the _user_cache? isn't it enough to check if the user is in the db, and if not, insert it?